cmd/internal/cli/singularity.go

```diff
@@ -188,6 +188,22 @@ func setSylogColor() {
 	}
 }
 
+// createDataDir will create ~/.singularity if it does not already exist
+func createDataDir(usr *user.User) {
+	dataDir := path.Join(usr.HomeDir, ".singularity")
+
+	if _, err := os.Stat(dataDir); err != nil {
+		if os.IsNotExist(err) {
+			sylog.Verbosef("%s does not exist. Creating.", dataDir)
+			os.Mkdir(dataDir, os.ModePerm)
+		} else {
+			sylog.Fatalf("Error attempting to stat %s: %s\n", dataDir, err)
+		}
+	} else {
+		sylog.Verbosef("%s already exits. Not creating.", dataDir)
```

Comment:
Can we use `Debugf` here instead?

```suggestion
		sylog.Debugf("%s already exits. Not creating.", dataDir)
```